tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -162,20 +176,11 @@ def pre_build_jobspecs(self):
     return []
 
   def build_jobspec(self):
-    if self.platform == 'windows':
-      raise Exception("Not supported yet")
-    else:
-      if self.platform == 'linux':
-        environ = {}
-        if self.arch == 'x86':
-          environ['SETARCH_CMD'] = 'linux32'
-        return create_docker_jobspec(self.name,
```

Comment:
I'm a little confused since currently it looks like docker <i>is</i> nested for this target? The change looks right to me though